Jenkinsfile

```diff
@@ -261,7 +261,21 @@ def k8sTest(Map args = [:]) {
 }
 
 /**
-* This method runs the packaging
+* This method runs the packaging for ARM
+*/
+def packagingArm(Map args = [:]) {
+  PACKAGES = "docker"
```

Comment:
We can probably drop this. This environment variable is needed when beats are released in the RM to make sure we do not produce extra artifacts.

Reply:
Thanks @kvch , I just changed it.

OTOH, the existing packaging automation it's using that particular flag:
- https://github.com/elastic/beats/blob/125c29d28e1cf718aac4d8589406097707819458/.ci/packaging.groovy#L227

Should it be removed too?

Eventually this PR should be the only way to build all the packages, therefore the `.ci/packaging.groovy` will be removed, but for the time being we are having both packaging processes, even though if they are duplicated. Since we don't wanna change the current process yet until it has been fully tested.

Reply:
@kvch , it seems the `PACKAGES="docker"` is needed otherwise:

```
[2021-02-16T13:12:36.923Z] standard_init_linux.go:211: exec user process caused "exec format error"
[2021-02-16T13:12:36.923Z] standard_init_linux.go:211: exec user process caused "exec format error"
[2021-02-16T13:13:48.030Z] package ran for 5m23.440051467s
[2021-02-16T13:13:48.031Z] Error: failed building packetbeat-oss type=deb for platform=linux/arm64: failed while running FPM in docker: running "docker run -e EXEC_UID=1152 -e EXEC_GID=1152 --rm -w /app -v /var/lib/jenkins/workspace/Beats_beats_PR-24041/src/github.com/elastic/beats/packetbeat:/app docker.elastic.co/beats-dev/fpm:1.11.0 fpm --force --input-type tar --output-type deb --name packetbeat --architecture arm64 --version 8.0.0 --vendor Elastic --license ASL-2.0 --description Packetbeat analyzes network traffic and sends the data to Elasticsearch. --url https://www.elastic.co/products/beats/packetbeat --after-install build/package/packetbeat-oss-linux-arm64.deb/scripts/systemd-daemon-reload.sh --config-files /etc/packetbeat/packetbeat.yml -p build/distributions/packetbeat-oss-8.0.0-SNAPSHOT-arm64.deb build/distributions/tmp-deb-packetbeat-8.0.0-SNAPSHOT-linux-arm64-8205217749.tar.gz" failed with exit code 1
```

See https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/PR-24041/runs/2/steps/10864/log/?start=0

I'm just reverting my last commit and see if I see the same behaviour

Reply:
I'm afraid the `PACKAGES=docker` is required otherwise the above error will happen.

@kvch , any clue?

Reply:
Indeed. Building these artifacts on ARM needs a bit more work.